gravitee-apim-cypress/cypress/integration/e2e/ui-api-search.ts

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
this file should not be at the same level as cypress-config.json. create a dedicated package please

Reply:
Another subfolder for the actual tests you mean?
Can we not get rid of all these different cypress configurations? Most of their content is duplicated anyway, they have only very few differences. We could maybe cover these differences in the npm run scripts using command-line parameter, what do you think?

Reply:
As discussed, this will be covered by next change that I'll start today.